### PR TITLE
UHM-7163: Enable MFA on all PIH EMR instances

### PR DESCRIPTION
**Pull request by mogoodrich**

### CODE REVIEW

**Comment by mogoodrich** [comment]
@mseaton  does this look correct?  I just moved the authentication config we had set up on HUM-CI to the top level configs, and removed totp (which I don't think we are using, and possibly doesn't work anyway?)

File: configuration/pih/pih-config-haiti.json [hunk 1]
```diff
@@ -15,6 +15,17 @@
         "registrationapp.paperRecordIdentifierDefinition": "paperrecord.patientDataDefinition.paperRecordIdentifier"
     },
 
+    "authenticationConfig": {
+      "scheme": "2fa",
+      "schemes": {
+        "2fa": {
+          "config": {
+            "secondaryOptions": "secret"
```
Comment (mseaton) [inline configuration/pih/pih-config-haiti.json:23]: totp should also be added, unless there is a specific reason we are choosing not to include it @mogoodrich ?  why do you think it doesn't work?

File: configuration/pih/pih-config-mirebalais.json [hunk 2]
```diff
@@ -15,6 +15,17 @@
     "registrationapp.paperRecordIdentifierDefinition": "paperrecord.patientDataDefinition.paperRecordIdentifier"
   },
 
+  "authenticationConfig": {
+    "scheme": "2fa",
+    "schemes": {
+      "2fa": {
+        "config": {
+          "secondaryOptions": "secret"
```
Comment (mseaton) [inline configuration/pih/pih-config-mirebalais.json:23]: Same here.

**Review by mseaton — CHANGES_REQUESTED** [review]

File: configuration/pih/pih-config-haiti.json (hunk 1 above)
Comment (mogoodrich) [reply configuration/pih/pih-config-haiti.json:23]: Perfect, I'll add it back it... no reason to think it doesn't work, I just had forgotten if we had built it or not.

**Comment by mogoodrich** [comment]
Added totp back in, merging, thanks @mseaton

**Pull request merged**